apis/src/components/organisms/standings.rs

```diff
@@ -1,67 +1,57 @@
+use std::collections::HashMap;
+
 use crate::components::molecules::score_row::ScoreRow;
 use crate::responses::TournamentResponse;
 use leptos::*;
-use uuid::Uuid;
+use shared_types::Tiebreaker;
+const TH_CLASS: &str = "py-1 px-1 md:py-2 md:px-2 lg:px-3 font-bold uppercase";
 
 #[component]
 pub fn Standings(tournament: Signal<TournamentResponse>) -> impl IntoView {
-    let th_class = "py-1 px-1 md:py-2 md:px-2 lg:px-3 font-bold uppercase";
+    let tiebreakers_view = tournament
+        .get_untracked()
+        .tiebreakers
+        .iter()
+        .map(|tiebreaker| {
+            view! { <th class=TH_CLASS>{tiebreaker.pretty_str().to_owned()}</th> }
+        })
+        .collect_view();
     view! {
         <table class="m-2 table-fixed max-w-fit h-fit">
             <thead>
                 <tr>
-                    <th class=th_class>Pos</th>
-                    <th class=th_class>Player</th>
-                    {tournament()
-                        .tiebreakers
-                        .iter()
-                        .map(|tiebreaker| {
-                            view! { <th class=th_class>{tiebreaker.pretty_str().to_owned()}</th> }
-                        })
-                        .collect_view()}
+                    <th class=TH_CLASS>Pos</th>
+                    <th class=TH_CLASS>Player</th>
+                    {tiebreakers_view}
+                    <th class=TH_CLASS>Finished</th>
                 </tr>
             </thead>
             <tbody>
                 <For
-                    each=move || { tournament().standings.results().into_iter() }
-                    key=|players_at_position| {
-                        players_at_position.iter().map(|(uuid, _, _)| *uuid).collect::<Vec<Uuid>>()
-                    }
-
-                    let:players_at_position
+                    each=move || tournament().standings.results().into_iter().flatten()
+                    key=|(uuid, position, finished, hash)| (
+                        *uuid,
+                        position.clone(),
+                        *finished,
+                        hash.values().map(|x| x.ceil() as i64).sum::<i64>(),
```

Comment:
This was a sort of hacky solution to fix the reactivity but it seems to work correctly. the values in the map are f32 but I couldn't sum over f32 because f32 doesn't implement hash and the key= requires hahsable elements. When Score row gets called it uses the initial values in the hash

Reply:
I dont get the use of celi here, why not just `hash.values().sum::<f32>() as i64` ?